src/index.ts

```diff
@@ -34,6 +35,8 @@ router.get("/", async () => {
 
 router.patch("/guild/member", changeNickname);
 
+router.delete("/guildroles/:roleId", deleteGuildRoleHandler);
```

Comment:
1. Is there any particular reason for using `guildroles` in the endpoint instead of `roles` (as in other role related endpoints in this file)?
2. Do we need an authorisation check for superuser here?

Reply:
1. We have an endpoint DELETE /roles which deletes role from user. So, I find DELETE /guildroles/:roleId more suitable for the purpose.
2. The authorisation check for super user will be done from website backend as it is going to hit from there only.

Reply:
Ok. Got it

Reply:
@pankajjs But the existing endpoint is just `/roles` and your endpoint would be `/roles/:roleId`. Right?

Reply:
Thanks for pointing it out. It would definitely work.
I have made the changes.